tokens.tf

```diff
@@ -0,0 +1,49 @@
+#
+# @NOTE: At the moment we use the token webhook bridge to manage our user tokens, but we need a user to manage
+# tokens.csv content; this will be done by the platform hub. This user has to be managed for obvious reasons.
+# Hopefully once we've come up with a better implementation we can drop this requirement. Note, the following is
+# ONLY done if the module is managing the state bucket
+#
+
+# The policy used by the assets user
+data "aws_iam_policy_document" "assets_policy" {
+  statement {
+    actions = ["s3:*"]
```

Comment:
Is it safe to leave the actions this open or can it be tightened down? Will there just be Put / Get requests on a specific resource?